packages/thirdweb/src/transaction/actions/to-serializable-transaction.ts

```diff
@@ -55,7 +57,12 @@ export async function toSerializableTransaction(
     );
     const { gas, maxFeePerGas, maxPriorityFeePerGas } = await getZkGasFees({
       transaction: options.transaction,
-      from: options.from ? getAddress(options.from) : undefined,
+      from:
+        typeof options.from === "string" // Is this just an address?
+          ? checksumAddress(options.from)
```

Comment:
Getaddress is nice because it throws if invalid input, not sure if checksum does that